CHANGELOG.md

```diff
@@ -1,25 +1,31 @@
 # Cromwell Change Log
 
-## 29
+## 29 Release Notes
 
 ### Breaking Changes
 
-* Request timeouts for HTTP requests on the REST API now return a 503 status code instead of 500. The response for a request timeout is no longer in JSON format.
-* The metadata endpoint no longer returns gzipped responses by default. This now needs to be explicitly requested with an `Accept-Encoding: gzip` header 
+* **Command line**  
+In preparation for supporting CWL scripts (yes, you read that right!), we have extensively revised the Command Line in Cromwell 29. For more details about the usage changes, please see the
+[README](https://github.com/broadinstitute/cromwell#command-line-usage). And stay tuned to the [WDL/Cromwell blog](https://software.broadinstitute.org/wdl/blog) over the next couple of months for more news about CWL.
```

Comment:
"over the coming months" reads almost the same without quite the same degree of potentially regrettable specificity